keyboards/knobgoblin/info.json

```diff
@@ -8,32 +8,33 @@
         "LAYOUT_ortho": {
             "layout": [
                 
-				{"x": 1, "y": 0},
-                {"x": 2, "y": 0},
-                {"x": 3, "y": 0},
-                {"x": 4, "y": 0},
+				        {"Label": "F17", "x": 1, "y": 0},
+                {"Label": "F18", "x": 2, "y": 0},
+                {"Label": "F19", "x": 3, "y": 0},
+                {"Label": "F20", "x": 4, "y": 0},
```

Comment:
Is the wonky indentation intentional? (I think the project indentation policy is 4 space, no hard-tabs.) If not could you filter this through something like `underscore print --wrapwidth 72` please? (`jq` doesn't produce quite as nice-looking output unfortunately.) See https://github.com/ddopson/underscore-cli